pkg/services/ngalert/provisioning/notification_policies_test.go

```diff
@@ -158,7 +158,7 @@ func TestNotificationPolicyService(t *testing.T) {
 
 		updated, err := sut.GetPolicyTree(context.Background(), 1)
 		require.NoError(t, err)
-		require.Equal(t, models.ProvenanceAPI, updated.Provenance)
+		require.Equal(t, definitions.ProvenanceAPI, updated.Provenance)
```

Comment:
(just noting for readers)

Discussed for some time offline about this change and other `models->definitions` changes. The code being tested already works in terms of objects in `definitions`, and this ends up being OK in the long run.